### DISCUSSION TITLE
Studio One (Win 11) crashing

### DISCUSSION THREAD

**Discussion by yoyesiku** [DISTRHO/Cardinal#604] [Q&A]
Anyone have Cardinal running in Studio One 6, latest build? I've tried installing Cardinal 3 separate times and each time I tried to get it running in Studio One, it crashes. The first time, I was able to load Cardinal into a track, but the moment I resized the Cardinal window inside Studio One, it crashed and I could get my song loaded again with the Cardinal track (had to go back to a version of the song without the Cardinal track). 2nd and third time installing, Studio One crashes when trying to load Cardinal into a track. Any advice on what could be causing the issue?

I'm running the latest build of Cardinal (23.10)
Studio One 6.5.1.96553

System Info:
OS Name	Microsoft Windows 11 Pro
Version	10.0.22621 Build 22621
Processor	AMD Ryzen Threadripper 3970X 32-Core Processor, 3701 Mhz, 32 Core(s), 64 Logical Processor(s)
BIOS Version/Date	American Megatrends Inc. 1603, 3/4/2022
SMBIOS Version	3.2
Embedded Controller Version	255.255
BIOS Mode	UEFI
BaseBoard Manufacturer	ASUSTeK COMPUTER INC.
BaseBoard Product	ROG ZENITH II EXTREME ALPHA
BaseBoard Version	Rev 1.xx
Secure Boot State	Off
Installed Physical Memory (RAM)	64.0 GB
GPU: RTX 3080

**Comment by dromer** [comment]
Is this with VST2, VST3 or both?

**Comment by yoyesiku** [comment]
Just the VST3.

**Comment by falkTX** [comment]
same issue as https://github.com/DISTRHO/Cardinal/issues/441 and https://github.com/AidaDSP/AIDA-X/issues/22

**Comment by yoyesiku** [comment]
Didn't see it in my search for Studio One. I'll delete this post. Thanks.
